app/src/main/res/layout-land/walkthrough_welcome_fragment.xml

```diff
@@ -52,7 +52,7 @@
       android:layout_width="180dp"
       android:layout_height="180dp"
       android:layout_marginStart="56dp"
-      android:src="@drawable/ic_onboarding_0"
+      android:src="@drawable/ic_landscape_onboarding_0"
```

Comment:
Ditto as above

Reply:
@SayantanBanerjee16 
will create a PR to fix this

Reply:
@nikitamarysolomanpvt Okay am reverting from this PR. Shall I go ahead creating a separate PR for to fix that?

Reply:
@SayantanBanerjee16 You may update your code with develop now.

Reply:
Done